www/views/page/content/list.php

```diff
@@ -23,30 +27,49 @@ function cap($text, $len)
 }
 
 if ($role > 1) {
-    $filter = isset($_GET['filter']) ? intval($_GET['filter']) : 1;
     /* 0 - alle stukjes
      * 1 - alle stukjes die klaar zijn
      * 2 - alle stukjes die klaar zijn & nog niet nagekeken
      */
+    $filter = isset($_GET['filter']) ? intval($_GET['filter']) : 1;
     echo "<div class='w-100 text-right'>";
-    echo "<a class='" . ($filter == 0 ? 'text-success' : '') . "' href='?action=list&filter=0'>alles</a> | ";
-    echo "<a class='" . ($filter == 1 ? 'text-success' : '') . "' href='?action=list&filter=1'>klaar</a> | ";
-    if ($role == 2) {
-        echo "<a class='" . ($filter == 2 ? 'text-success' : '') . "' href='?action=list&filter=2'>klaar & kan ik nakijken</a> ";
+    if ($action == "list"){
+        echo "<a class='" . ($filter == 0 ? 'text-success' : '') . "' href='?action=list&filter=0'>alles</a> | ";
+        echo "<a class='" . ($filter == 1 ? 'text-success' : '') . "' href='?action=list&filter=1'>klaar</a> | ";
+        if ($role == 2) {
+            echo "<a class='" . ($filter == 2 ? 'text-success' : '') . "' href='?action=list&filter=2'>klaar & kan ik nakijken</a> ";
+        }
+        if ($role == 3) {
+            echo "<a class='" . ($filter == 3 ? 'text-success' : '') . "' href='?action=list&filter=3'>klaar & nagekeken</a> | ";
+        }
     }
     if ($role == 3) {
-        echo "<a class='" . ($filter == 3 ? 'text-success' : '') . "' href='?action=list&filter=3'>klaar & nagekeken</a> | ";
-        echo "<a class='" . (!empty($catFilter) ? 'text-success' : '') ."' href='?action=list&filter=$filter&filtercat=1'>Filter op categorie</a>";
+        echo "<a class='" . (!empty($catFilter) ? 'text-success' : '') ."' href='?action=$action&filter=$filter&filtercat=1'>Filter op categorie</a>";
     }
     echo "</div>\n";
 }
+
+if ($role == 1) {
+    echo "<div class='w-100 text-right'>";
+    $filtercat = isset($_GET['filtercat']) ? intval($_GET['filtercat']) : 0;
```

Comment:
Verwarrend dat `filtercat` hier heel iets anders doet dan in beheerder-modus. Eigenlijk kan het voor de beheerder-modus beter `show_filter_options` heten ofzo.